### DISCUSSION TITLE
wireguard使用一段时间后会失效，必需reboot系统才能恢复，是什么原因？

### DISCUSSION THREAD

**Discussion by zenwars** [XTLS/Xray-core#3541] [Q&A]
使用superviosr启动的xray，每天使用superviosrctl restart xray来重启

superviosr配置：
```
[program:xray]
command=xray run -c /etc/xray/config.json
autostart=true
autorestart=true
user=root
```
wireguard是通过wgcf-cli生成的，能正常用，reboot后也能用，就是用了一段时间（或者restart后）就失效了，路由后不走wireguard的请求能正常工作，请问是什么问题？

Xray v1.8.16

**Comment by ZqinKing** [comment]
有类型的情况。XRAY服务端部分流量使用warp的wireguard进行出站。每次重启XRAY后，抓包wg0接口有几率出现只有发送的报文，没有回包的现象（浏览器表象就是相关访问不通）。多次重启XRAY后可以使wg0接口恢复正常。
